docs/data_access.rst

```diff
@@ -0,0 +1,218 @@
+:doc:`contribute to the project <CONTRIBUTING>` please give it a shot!
+
+.. _access-datapackage:
+
+---------------------------------------------------------------------------------------
+Data Packages
+---------------------------------------------------------------------------------------
+
+.. note::
+
+    Prior to v0.4.0 of PUDL we only published processed data as `tabular data
+    packages <https://frictionlessdata.io/specs/tabular-data-package/>`__. As of
+    v0.4.0 we are will distribute the SQLite databases and Apache Parquet files
+    alongside a set of data packages. As of PUDL v0.5.0 we will be generating SQLite
+    and Apache Parquet outputs directly, and will no longer be archiving tabular data
+    packages as the format of record, and the format conversions described below will
+    no longer be necessary.
+
+Archived Data Packages
+^^^^^^^^^^^^^^^^^^^^^^
+
+We periodically publish data packages containing the full outputs from the PUDL ETL
+pipeline on `Zenodo <https://zenodo.org>`__, and open data archiving service provided
+by CERN. The most recent release can always be found through this concept DOI:
+`10.5281/zenodo.3653158 <https://doi.org/10.5281/zenodo.3653158>`__. Each individual
+version of the data releases will be assigned its own unique DOI.
+
+All of our archived products can be found in the `Catalyst Cooperative Community on
+Zenodo <https://zenodo.org/communities/catalyst-cooperative/>`__. These archives and
+the DOIs associated with them should be permanently accessible, and are suitable for
+use as references in academic and other publications.
+
+Once you've downloaded or generated your own tabular data packages you will probably
+want to convert them into a more analysis oriented file format. We typically use
+SQLite for the core FERC and EIA data, and Apache Parquet files for the very long
+tables like EPA CEMS.
+
+Converting to SQLite
+^^^^^^^^^^^^^^^^^^^^
+
+If you want to access the data via SQL, we have provided a script that loads several
+data packages into a local :mod:`sqlite3` database. Note that these data packages
+**must** have all been generated by the **same** ETL run, or they will be considered
+incompatible by the script. For example, to load three data packages generated by our
+example ETL configuration into your local SQLite DB, you could run the following
+command from within your PUDL workspace:
+
+.. code-block:: console
+
+    $ datapkg_to_sqlite \
+        datapkg/pudl-example/ferc1-example/datapackage.json \
+        datapkg/pudl-example/eia-example/datapackage.json \
+
+Run ``datapkg_to_sqlite --help`` for more details.
+
+Converting to Apache Parquet
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :doc:`data_sources/epacems` data approaches 100 GB in size uncompressed, which is
+too large to work with directly in memory on most systems, and take a very long time
+to load into SQLite. Instead, we recommend converting the Hourly Emissions table into
```

Comment:
I would just end the sentence after "too large to work with on most systems" and skip the bit about directly in memory.